components/camel-graphql/src/main/java/org/apache/camel/component/graphql/GraphqlComponent.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.camel.component.graphql;
+
+import java.util.Map;
+
+import org.apache.camel.Endpoint;
+import org.apache.camel.spi.annotations.Component;
+import org.apache.camel.support.DefaultComponent;
+
+@Component("graphql")
+public class GraphqlComponent extends DefaultComponent {
+
+    @Override
```

Comment:
Are you planning let the component configurable in the component level? Then will be nice to add the GraphqlConfiguration as private field in the class

Reply:
Sorry. I didn't understand your comment. Can you give me an example?

Reply:
I've decided to remore GraphqlConfiguration and define the parameters in GraphqlEndpoint for now.

Reply:
Please review the code again.